src/screens/TopicPageScreen/elements/ButtonFollowing.js

```diff
@@ -22,14 +22,14 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     paddingVertical: 6,
     borderWidth: 1,
-    borderColor: colors.bondi_blue,
+    borderColor: COLORS.anon_primary,
     borderRadius: 8
   },
   textButtonFollowing: {
     fontFamily: fonts.inter[600],
     fontWeight: 'bold',
     fontSize: normalizeFontSize(12),
-    color: colors.bondi_blue
+    color: COLORS.anon_primary
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color scheme has been changed from `colors.bondi_blue` to `COLORS.anon_primary`. Ensure that this change is intentional and consistent with the overall application's color theme. If `COLORS.anon_primary` is not defined in the `COLORS` object, it will cause an error.
```diff
-    borderColor: colors.bondi_blue,
+    borderColor: COLORS.anon_primary,

-    color: colors.bondi_blue
+    color: COLORS.anon_primary
```
Please verify these changes with your design team or refer to your design guidelines to ensure consistency across the application.


<!-- This is an auto-generated comment by OSS CodeRabbit -->